integrationtest/backend/tck/src/main/java/org/hibernate/search/integrationtest/backend/tck/search/predicate/ExistsPredicateObjectsSpecificsIT.java

```diff
@@ -159,19 +165,19 @@ public void nested_multiIndexes_wrongStructure() {
 
 	@Test
 	public void nested_multiIndexes_differentFields() {
-		assumeFullMultiIndexCompatibilityCheck();
-		SearchPredicateFactory f = mainIndex.createScope( differentFieldsIndex ).predicate();
 		String fieldPath = "nested";
 
-		assertThatThrownBy( () -> f.exists().field( fieldPath ) )
-				.isInstanceOf( SearchException.class )
-				.hasMessageContainingAll(
-						"Inconsistent configuration for field '" + fieldPath + "' in a search query across multiple indexes",
-						"Attribute 'staticChildren' differs" )
-				.satisfies( FailureReportUtils.hasContext(
-						EventContexts.fromIndexNames( mainIndex.name(), differentFieldsIndex.name() ),
-						EventContexts.fromIndexFieldAbsolutePath( fieldPath )
-				) );
+		assertThatQuery( mainIndex.createScope( differentFieldsIndex ).query()
+				.where( f -> f.exists().field( fieldPath ) ) )
+				.hasDocRefHitsAnyOrder( mainIndex.typeName(), DOCUMENT_3 );
+
+		assertThatQuery( mainIndex.createScope( noInnerNestedField ).query()
+				.where( f -> f.exists().field( fieldPath ) ) )
+				.hasDocRefHitsAnyOrder( mainIndex.typeName(), DOCUMENT_3 );
+
+		assertThatQuery( mainIndex.createScope( differentInnerNestedField ).query()
+				.where( f -> f.exists().field( fieldPath ) ) )
+				.hasDocRefHitsAnyOrder( mainIndex.typeName(), DOCUMENT_3 );
```

Comment:
These are nice, but in order to test if this really works, we need at least one matching and one non-matching document in each index.

Reply:
That's what I thought 😃, but then looked at the previous tests and was hesitant as those were not adding any documents to "other" indices

Reply:
Right, that's because there's no point adding documents to other indices if search will fail and the documents won't be returned anyway :)